cmd/compute-domain-controller/controller.go

```diff
@@ -68,6 +71,7 @@ func (c *Controller) Run(ctx context.Context) error {
 		imageName:       c.config.flags.imageName,
 		clientsets:      c.config.clientsets,
 		workQueue:       workQueue,
+		chartName:       c.config.flags.chartName,
```

Comment:
as mentioend before -- please move this to the top so that the vars are set in the same order they exist in the struct definition

Reply:
moved